### PR TITLE
Tweak tooltip & policy card components

### PR DESCRIPTION
**Pull request by cisacke**

### CODE REVIEW

**Comment by cisacke** [comment]
@danielnovograd @jmcolella this has the changes to `Tooltip`

File: src/atoms/Tooltip/index.js [hunk 1]
```diff
@@ -46,12 +48,13 @@ class Tooltip extends React.Component {
...
         )}
       >
-        <Icon icon='questionMark' className={styles['tooltip']} />
+        { text }
         <span
           className={classnames(
             children && styles['hover-message'],
             left && styles['left'],
             right && styles['right'],
+            hoverMessageClassName && hoverMessageClassName
```
Comment (danielnovograd) [inline src/atoms/Tooltip/index.js]: If `classnames` pulls out any `undefined` values, could this be just `hoverMessageClassName` without the short-circuiting?

File: src/organisms/cards/PolicyCard/Readme.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
       customerService='A+'
       totalCustomers={5200000}
       policyType='Simplified Issue Whole Life'
+      onPolicyTypeHover={() => {}}
```
Comment (danielnovograd) [inline src/organisms/cards/PolicyCard/Readme.md]: do we need this?

File: src/atoms/Tooltip/index.js (hunk 1 above)
Comment (cisacke) [reply src/atoms/Tooltip/index.js]: ahh, didn't know classnames did that, good call

File: src/organisms/cards/PolicyCard/Readme.md (hunk 2 above)
Comment (cisacke) [reply src/organisms/cards/PolicyCard/Readme.md]: nope, good catch

**Pull request merged**